### PR TITLE
Hindi Category in Kubernetes Discussion

### PR DESCRIPTION
**Pull request by jayeshmahajan**
**Hindi Category in Kubernetes Discussion**

**Which issue(s) this PR fixes**:

Fixes #
IT Update details on Hindi Category in Kubernetes Discussion

### CODE REVIEW

**Review by jayeshsub — APPROVED** [review]

File: communication/moderators.md [hunk 1]
```diff
@@ -229,6 +236,7 @@ Administrators seats: 6
 [github administration team]: https://git.k8s.io/community/github-management#github-administration-team
 [trust system]: https://blog.discourse.org/2018/06/understanding-discourse-trust-levels/
 [chinese]: https://discuss.kubernetes.io/t/about-the-chinese-category/2881
+[hindi]: https://discuss.kubernetes.io/c/regional-discussions/hindi/29
```
Comment (jberkus) [inline communication/moderators.md:239]: Fix link:

```suggestion
[hindi]: https://discuss.kubernetes.io/t/about-the-hindi-category/30051
```

**Review by jberkus — CHANGES_REQUESTED** [review]
One change needed.

File: communication/moderators.md (hunk 1 above)
Comment (jayeshmahajan) [reply communication/moderators.md]: Done

**Review by MadhavJivrajani — APPROVED** [review]
/approve
Thank you!

**Pull request merged**